src/AvrNTPClient.cpp

```diff
@@ -420,7 +420,7 @@ boolean NTPClient::setInterval(int shortInterval, int longInterval) {
 
 boolean NTPClient::setTimeZone(int timeZone)
 {
-	if (timeZone >= -11 || timeZone <= 13) {
```

Comment:
This was right. Here: https://en.wikipedia.org/wiki/Time_zone#/media/File:Solar_time_vs_standard_time.png you can see there are time zones between -12 and +14